modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc

```diff
@@ -27,21 +27,62 @@ just like your other ownCloud files and folders.
 
 Compared to standard SMB access, WND has advanced features like:
 
-* User lockout prevention and password reset
-* More authentication mechanisms against the backend
-* Change notifications
-* Enhanced ACL support
+. User lockout prevention and password reset
+. More authentication mechanisms against the backend
+. Listen to change information triggered by the backend
+. Enhanced ACL support
+. Collaborative WND (CWND)
 
-Here is a brief description of the advanced features:
+.Brief Description of Advanced Features:
 
+User lockout prevention and password reset::
 Depending on the Windows or Samba policy, the user could get locked out of his account if he enters a wrong password a number of times. The lockout prevention tries to avoid this from happening, by resetting the password if it is wrong. For the case of ownCloud's standard SMB connector, the password won't be reset. It could happen, that the user gets locked out of the file server.
 
+More authentication mechanisms against the backend::
 Please see the details about the xref:enterprise/external_storage/enterprise_only_auth.adoc[Enterprise-Only Authentication Options]
 
-WND has the ability to trigger change notifications to users based on changes for files and folders. 
+Listen to change information triggered by the backend::
+Native Windows File Servers provide the ability to proactively send change notifications made to files and folders somewhere in a share. This mechanism then updates the ownCloud database and provides the changes made to accessing users. Users do not need to manually check for changes in all possible locations of their mount. Changes processed are also propagated to sync clients automatically.
```

Comment:
I think samba can do that as long as all the accesses are done through the SMB protocol. This is different than windows, where you get notifications even if the changes are done directly in the storage.
Taking into account the requirements, we might want to skip this information from here. It won't be brief otherwise.

Reply:
Do a text change suggestion to satisfy both native and samba file servers. This would help a lot.

Reply:
Maybe adding the information as a note is better. The requirements are simply different

Reply:
We have in all items a description of what it is about. We cant just drop a "note" here. We need a proper explaining text. Suggestions are really welcomed.

Reply:
"Samba can send file notifications as well, as long as all the file actions are performed through the SMB protocol. However, it won't work if the action is performed directly inside the FS used by Samba"

Reply:
fixed